src/qcronfield.cpp

```diff
@@ -81,11 +81,22 @@ QCronField::
 _parseEvery(QString & str)
 {
     str.remove(0, 1);
+    uint offset = 0;
     if (!_last_node) {
         throw QCronFieldException(QString("Every node for string %1 has null last node.")
                                       .arg(str));
     }
-    return new QCronEveryNode(_last_node, _parseInt(str));
+    else if(dynamic_cast<QCronRangeNode *>(_last_node) == nullptr
```

Comment:
Nit: no need for `else` here (and below). If the condition is true, the code will throw an exception, which will skip the rest of the function.